src/server/game/AuctionHouse/AuctionHouseMgr.cpp

```diff
@@ -153,9 +153,16 @@ void AuctionHouseMgr::SendAuctionSalePendingMail(AuctionEntry* auction, SQLTrans
     if (owner || owner_accId)
     {
         sScriptMgr->OnBeforeAuctionHouseMgrSendAuctionSalePendingMail(this, auction, owner, owner_accId, sendMail);
+
+        uint32 deliveryDelay = sWorld->getIntConfig(CONFIG_MAIL_DELIVERY_DELAY);
```

Comment:
I think that you can set `auto` instead of uint32, because the function `getIntConfig()` already return an uint32 by default